source/vistas/ui/controls/float_ctrl.py

```diff
@@ -6,6 +6,8 @@
 
 
 class FloatCtrlEvent(FloatCtrlEventBase):
+    """ An event that simply passes a float value. """
```

Comment:
The value it passes seems less important that what it relates to (really, `value` could be anything). In this case, it's relationship with `FloatCtrl` is more useful information than that floats will be passed to it.